themes/classic/_dev/css/partials/_variables.scss

```diff
@@ -1,6 +1,8 @@
 $gray-darker: #232323;
+$gray-searchbar: #737373;
```

Comment:
What if we use this color for something different than a searchbar? :thinking:

Reply:
I guess that's the same thing for using "$btn-primary-color: #fff;" for something different than a button